pkg/swarm/swarm.go

```diff
@@ -28,6 +28,9 @@ const (
 	ExtendedPO        uint8 = MaxPO + 5
 	MaxBins                 = MaxPO + 1
 	ChunkWithSpanSize       = ChunkSize + SpanSize
+	SignatureSize           = 65
```

Comment:
these are all related to `soc` so they should be prefixed accordingly. the min and max chunk size are specific to socs only, not to content addressed chunks. and maybe worth to comment that this is the "chunk size" with sharky, because sharky cares only about the max size of a chunk you could at most get (and if the data is too large it would err on `Write` call)